docs/en/04_Changelogs/4.11.0.md

```diff
@@ -21,6 +23,21 @@ In accordance with our [PHP support policy](/Getting_Started/Server_Requirements
 
 ## Features and enhancements {#features-and-enhancements}
 
+### Preview any DataObject in any admin section {#cms-preview}
+
+The CMS preview panel has historically on been available on `Versioned` `DataObjects` in the Pages admin section. This has now been expanded to allow any `DataObject`(regardless of whethwr they jse the`Versioned` extension) to be previewed in any section of the CMS.
+
+This can be used to allow content authors to see the content they are creating in the context it will be presented to users. Example use cases include previewing `DataObjects` which belong to a page (e.g. the [dnadesign/silverstripe-elemental module](https://github.com/silverstripe/silverstripe-elemental) allows previewing elemental blocks which are not inline-editable), and previewing `DataObjects` in a custom admin section, such as email or system-generated PDF templates.
```

Comment:
```suggestion
This can be used to allow content authors to see the content they are creating in the context it will be presented to users. Example use cases include previewing `DataObjects` which belong to a page (e.g. the [dnadesign/silverstripe-elemental module](https://github.com/silverstripe/silverstripe-elemental) allows previewing elemental blocks which are not inline-editable), and previewing `DataObjects` in a custom admin section, such as templates for emails or system-generated PDF documents.
```